linera-base/src/crypto/secp256k1/evm.rs

```diff
@@ -399,6 +409,15 @@ impl EvmSignature {
         self.verify_inner::<T>(prehash, author)
     }
 
+    /// Checks a signature against a recovered public key.
+    pub fn check_with_recover<'de, T>(&self, value: &T) -> Result<(), CryptoError>
```

Comment:
This should take the address as argument and verify that the `public_key` matches.

Reply:
Not necessary with https://github.com/linera-io/linera-protocol/pull/4088/commits/ac549502476344ed4917c72ba79aa716644a5a5a